command/server/init.go

```diff
@@ -316,6 +316,7 @@ func (p *serverParams) initJSONRPCAddress() error {
 func (p *serverParams) initDataFeedMQAddress() error {
```

Comment:
I would just rename this function to `initDataFeedParams()` since the verifyOutcomeApiUrl doesn't really have anything to do with the MQ